README.md

```diff
@@ -10,7 +10,12 @@ about external nodes.
 
 ## Prerequisites
 
-Consul ESM requires at least version 1.0.1 of Consul.
+Consul ESM requires at least version 1.4.1 of Consul.
+
+| ESM version | Consul version required |
+|-------------|-------------------------|
+| 0.3.2 | 1.4.1 |
```

Comment:
We can probably do 1.4.1+ here, pretty clear but just to be sure there isn't confusion.